1-exercises/E-array-map/exercise.js

```diff
@@ -2,3 +2,16 @@
 // Write multiple solutions using different syntax (as shown in the README)
 
 var numbers = [0.1, 0.2, 0.3, 0.4, 0.5];
+// first method
+const timesHundred = numbers.map(function mutipliedHundred(num) {
+  return num * 100;
+});
+console.log(timesHundred);
+// 2nd method
+const multipleOfHundred = numbers.map((number) => number * 100);
+console.log(multipleOfHundred);
+// 3rd method
+const multiHundred = numbers.map((num) => {
```

Comment:
so this is the same as method 3 insread you can do a forloop like this 
```suggestion
for(var i=0; i<numbers.length; i++) {
	numbers[i] = numbers[i] * 100;
}
console.log(numbers);
```